src/utils/format-numbers.js

```diff
@@ -25,12 +25,13 @@ import type { Microseconds, Milliseconds } from '../types/units';
 export function formatNumber(
   value: number,
   significantDigits: number = 2,
-  maxFractionalDigits: number = 3
+  maxFractionalDigits: number = 3,
+  style: string = 'decimal'
 ): string {
   /*
-   * Note that numDigitsOnLeft can be negative when the first non-zero digit
-   * is on the right of the decimal point.  0.01 = -1
-   */
+  * Note that numDigitsOnLeft can be negative when the first non-zero digit
+  * is on the right of the decimal point.  0.01 = -1
+  */
   const numDigitsOnLeft = Math.floor(Math.log10(Math.abs(value))) + 1;
```

Comment:
I realized something.
The value we pass for "percent" is something like `0.4` and not `40`. Therefore `numDigitsOnLeft` is too small and we need to add `2` here when `style` is percent. Something like this:

```js
  let numDigitsOnLeft = Math.floor(Math.log10(Math.abs(value))) + 1; 
  if (style === 'percent') {
    // We receive percent values as `0.4` but display them as `40`, so we
    // should add `2` here to account for this difference.
    numDigitsOnLeft += 2;
  }
  ...
```

Reply:
I believe this should fix the difference we see in the tests.

Reply:
```
<span
                        className="treeViewRowColumn treeViewFixedColumn totalTimePercent"
                        key="totalTimePercent"
    -                   title="100.0%"
    +                   title="100%"
                      >
    -                   100.0%
    +                   100%
                      </span>
                      <span
                        className="treeViewRowColumn treeViewFixedColumn totalTime"
                        key="totalTime"
                        title="1"
    @@ -226,13 +226,13 @@
                      }
                    >
```
Not working in this case.

Reply:
But this is actually better :) `100.0` was bad!

You can update the snapshots by running `yarn test -u`.